### PR TITLE
remove bright red for other color

### PR DESCRIPTION
**Pull request by makella**
for the qualitative ramp `Prism` I had accidentally added an additional color that was bright red. This bright red color was in turn being used to symbolize any feature that was considered `Other` in Builder. 

This PR fixes that problem.

Related to:
https://github.com/CartoDB/CartoColor/issues/21#event-885630345

### CODE REVIEW

**Comment by makella** [comment]
cc @xavijam

File: cartocolor.js [hunk 1]
```diff
@@ -1463,32 +1463,28 @@ var cartocolor = {
         "3": [
             "#5B3F95",
             "#1D6996",
-            "#129C63",
```
Comment (xavijam) [inline cartocolor.js:1466]: Hey but, why do we decide to remove one color in each size until 10?

Comment (xavijam) [reply cartocolor.js:1466]: In my opinion this should be tackled in another PR.

**Comment by rochoa** [comment]
@makella I guess after merging #19 and #27 we no longer need to merge this PR, right?

**Comment by makella** [comment]
right, we don't need! thanks @rochoa

**Pull request closed without merging**